scripts/gen_lib.lua

```diff
@@ -0,0 +1,71 @@
+lpeg = require('lpeg')
```

Comment:
Maybe that's better, but wouldn't `paths.lua` then be need to be found by some mechanism? At which point the mechanism can be used to find the paths directly... I guess I'm too used to languages where including code from the same directory as the code is a thing you just do...